src/index.js

```diff
@@ -0,0 +1,150 @@
+let allCityDetails
```

Comment:
you can add `;` at every line ends.
This is not mandatory in Js but we follow it as a standard

Reply:
eslint does not allow us to put semicolon at the end of each instruction

Reply:
Oh, then we can ignore this.